src/package.lisp

```diff
@@ -175,4 +175,7 @@
    #:defcvar
    #:get-var-pointer
    #:foreign-symbol-pointer
+   ;; *errno* and strerror are only available when the :cffi/errno ASDF system is loaded
```

Comment:
What are the cons of including *errno*/strerror directly without an auxiliar ASDF system?